slides/git-intro/index.html

```diff
@@ -0,0 +1,363 @@
+                    <p>Ever.</p>
+                </section>
+
+                <section>
+                    <h2>Git History - Why is it important?</h2>
+                    <ul>
+                        <li>Traceability - If something goes wrong, it's easy to trace down where, how and when.</li>
+                        <li>It helps with documentation - When documenting software, the Git history allows us to form a chronological view of the development process.</li>
+                        <li>All team members can follow progress easily.</li>
+                    </ul>
+                </section>
+
+                <section>
+                    <h2>Branching</h2>
+                    <p>Once you have a copy of the repository on your machine, the next thing to do before working on any features is create a <strong>branch</strong>.</p>
+                    <p>Branches are used to develop features isolated from each other.</p>
+                </section>
+
+                <section>
+                    <h2>Git Checkout from master</h2>
+                    <p>The master branch is the "default" branch when you create a repository. Use other branches for development and merge (more later) them back to the master branch upon completion.</p>
+                    <pre>
+                        <code class="sh" data-trim contenteditable>
+$ git checkout master
+$ git checkout -b tech103branch
+                        </code>
+                    </pre>
+                    <p>This creates a virtual copy of the master branch, but in a separate working tree, so your work is isolated from the master.</p>
+                </section>
+
+                <section>
+                    <h2>Git Checkout specific branch</h2>
+                    <p>Sometimes, you may need to make changes to a branch that hasn't yet been merged into master.</p>
+                    <p>Perhaps a pull request (...more later) needs modified, after you've started a new feature (on a new branch, of course....)</p>
+                    <p>In this situation, you can <strong>checkout</strong> the specific branch to work on</p>
+                    <pre>
+                        <code class="sh" data-trim contenteditable>
+                            $ git checkout hotfix_001
+                        </code>
+                    </pre>
+                </section>
+
+                <section>
+                    <h2>Staging</h2>
+                    <p>As you begin working on new features, you may find yourself adding new files to the repository.</p>
+                    <p>In this case, you are effectively 'staging' these files, before you <strong>commit</strong> (...more later) them to your branch.</p>
+                    <pre>
+                        <code class="zsh" data-trim contenteditable>
+                            $ git add 'filename'
```

Comment:
Can this be changed to a sample filename, e.g. git add index.html 

Mostly just so that people dont ask if they need the quote marks!